src/main/java/com/example/spring_bank_project/shared/infrastructure/adapter/publisher/RabbitMqReceiver.java

```diff
@@ -0,0 +1,19 @@
+package com.example.spring_bank_project.shared.infrastructure.adapter.publisher;
+
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.CountDownLatch;
+
+@Service
+public class RabbitMqReceiver {
+    private final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
jestes pewny ze wiesz jak działa i do czego służy CountDownLatch? dlaczego go tutaj uzyles?

Reply:
https://howtodoinjava.com/java/multi-threading/when-to-use-countdownlatch-java-concurrency-example-tutorial/ tu masz ladny obrazek co dobrze wyjasnia zagadnienie

Reply:
Ten kod to akurat copy paste, kiedy jeszcze chciałem użyć rabbita. Kod do wywalenia